### PR TITLE
Changed links of recourses, more beginner friendly

### PR DESCRIPTION
**Pull request by gijscor**
From the students I've gotten the signal that MDM is not helping them learn to code, as it is written for experienced developers and mostly just confuses them.  
Also the book "a simpler way to learn javaScript" is getting quite outdated (using var only e.g), and is a bit of a pain to read as it is a pdf file with hundreds of pages.

Many students have expressed to have had great help from the website javaScript.info, and I think it's a great webpage that is beginner friendly, but still manages to explain concepts well end in depth.

### CODE REVIEW

File: Week3/README.md [hunk 1]
```diff
@@ -10,26 +10,32 @@ In week three we will discuss the following topics:
...
 ### Here are resources that we like you to read as a preparation for the coming lecture:
 
 Please watch the following parts of the course, [Programming Foundations Fundamentals](https://www.lynda.com/Programming-Foundations-tutorials/Welcome/83603/90426-4.html) on Lynda.com (if you don't have access to Lynda yet ask Gijs):
+
     <br>4. Writing Conditional Code
     <br>5. Modular Code
     <br>6. Iteration: Writing Loops
     <br>7. More About Strings
     <br>8. Collections
     <br>11. When Things Go Wrong
```
Comment (remarcmij) [inline Week3/README.md]: Can you get rid of the`<br>` tags? They show up verbatim as this is a code block.

File: Week3/README.md [hunk 2]
```diff
@@ -10,26 +10,32 @@ In week three we will discuss the following topics:
...
-- 'Functions' of [_A Smarter Way To Learn JavaScript_](https://github.com/Sharique-Hasan/SaylaniBatch2-JavaScript/blob/master/A%20Smarter%20Way%20to%20Learn%20JavaScript.pdf) : Chapters 35 - 38
-- Functions ~ http://eloquentjavascript.net/03_functions.html
-- 'Objects' of [_A Smarter Way To Learn JavaScript_](https://github.com/Sharique-Hasan/SaylaniBatch2-JavaScript/blob/master/A%20Smarter%20Way%20to%20Learn%20JavaScript.pdf) : Chapters 69 - 75
-- 'Conditions' of [_A Smarter Way To Learn JavaScript_](https://github.com/Sharique-Hasan/SaylaniBatch2-JavaScript/blob/master/A%20Smarter%20Way%20to%20Learn%20JavaScript.pdf) : Chapters 10 - 14
-- Program structure ~ http://eloquentjavascript.net/02_program_structure.html
+## Read the following articles that are very important to programming!
+- [Array](http://javascript.info/array)
+- [Loops](http://javascript.info/while-for)
+- [Functions](http://javascript.info/function-basics)
+- [Functions](http://eloquentjavascript.net/03_functions.html)
```
Comment (remarcmij) [inline Week3/README.md]: Although these two Function links point to different URLs, in the formatted text you only see the word Functions repeated twice:

- Functions
- Functions

**Review by remarcmij — CHANGES_REQUESTED** [review]
Definitely an improvement. Just some formatting issues.

**Comment by gijscor** [comment]
@remarcmij formatting issues solved, thanks for the feedback.

**Comment by M3kH** [comment]
I'm in favour to moving away from `A smarter way of learning JavaScript`.

Since I guess was there because that was the book we started with; and by the time make-it sense.

But to be fair I would watch out a bit in just pull from there the references.

For now a 👍 on this.

File: Week3/README.md [hunk 3]
```diff
@@ -10,26 +10,32 @@ In week three we will discuss the following topics:
 • Functions
 ```
 
+
+> :boom: IMPORTANT: The concepts of these weeks are extremely important, and require very careful reading.
```
Comment (M3kH) [inline Week3/README.md:14]: `IMPORTANT` => repeats on => `are extremely important`

Comment (gijscor) [reply Week3/README.md:14]: I don't understand this

Comment (gijscor) [reply Week3/README.md:14]: it it ready to merge now?

**Review by remarcmij — APPROVED** [review]
LGTM, thanks Gijs.

**Pull request merged**